comp/core/sysprobeconfig/sysprobeconfigimpl/params.go

```diff
@@ -27,3 +30,10 @@ func WithSysProbeConfFilePath(confFilePath string) func(*Params) {
 		b.sysProbeConfFilePath = confFilePath
 	}
 }
+
+// WithRemoteConfigDirPath specifies the path to the remote config directory
+func WithRemoteConfigDirPath(remoteConfigDirPath string) func(*Params) {
+	return func(b *Params) {
+		b.remoteConfigDirPath = remoteConfigDirPath
+	}
+}
```

Comment:
Note: as the apply of that parameter requires changes of the `New` config method for sysprobe and has a broad impact on the agent as a whole, it'll be done in another PR.